research/delf/delf/python/training/README.md

```diff
@@ -118,11 +118,116 @@ files can take up to 12 hours and up to 500 GB of space disk.*
 
 ## Running the Training
 
-Assuming the TFRecord files were generated in the `gldv2_dataset/tfrecord/` directory, running 
-the following command should start training a model:
+For the training to converge faster, it is possible to initialize the ResNet backbone with the
+weights of a pretrained ImageNet model. The ImageNet checkpoint is available at the following
+location: [`http://storage.googleapis.com/delf/resnet50_imagenet_weights.tar.gz`](http://storage.googleapis.com/delf/resnet50_imagenet_weights.tar.gz).
+To download and unpack it run the following commands on a Linux box:
+```
+curl -Os http://storage.googleapis.com/delf/resnet50_imagenet_weights.tar.gz
+tar -xzvf resnet50_imagenet_weights.tar.gz
+```
 
+Assuming the TFRecord files were generated in the `gldv2_dataset/tfrecord/` directory, running 
+the following command should start training a model and output the results in the `gldv2_training`
+directory:
 ```
-python3 train.py \
+python train.py \
   --train_file_pattern=gldv2_dataset/tfrecord/train* \
   --validation_file_pattern=gldv2_dataset/tfrecord/validation*
+  --imagenet_checkpoint=resnet50_weights_tf_dim_ordering_tf_kernels_notop.h5 \
+  --dataset_version=gld_v2_clean \
+  --logdir=gldv2_training/
+```
+
+On a multi-GPU machine the batch size can be increased to speed up the training using the `--batch_size` parameter. On a 8 Tesla P100 GPUs machine you can set the batch size to `256`:
+```
+--batch_size=256
+```
+
+## Exporting the Trained Model
+
+Assuming the training output, the TensorFlow checkpoint, is in the `gldv2_training` directory,
+running the following command exports the model in the `gldv2_model` directory:
+```
+python model/export_model.py \
+  --ckpt_path=gs://dananghel-delf/training/internal/delf_weights \
```

Comment:
remove your gs:// path. I think this should instead be gldv2_training/delf_weights?

Reply:
Changed to python3.